homeassistant/components/zha/entity.py

```diff
@@ -175,6 +177,27 @@ def create_entity(
         """
         return cls(unique_id, zha_device, cluster_handlers, **kwargs)
 
+    def _init_from_quirks_metadata(self, entity_metadata: EntityMetadata) -> None:
```

Comment:
Should this also set `_attr_entity_category` to the `entity_type` of zigpy `EntityMetadata`?
Do note that some classes like `ZHASwitchConfigurationEntity` would overwrite that category again.